src/loaders/FileLoader.js

```diff
@@ -39,6 +41,19 @@ Object.assign( FileLoader.prototype, {
 
 		}
 
+		// If file is already in process of loading, wait for it to load.
+		if ( currentlyLoadingFiles[ url ] ) {
+
```

Comment:
Personally I prefer strict comparison with `undefined` for the consistent code in `FileLoader`.